.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 select = B,B9,BLK,C,D,DAR,E,F,I,W,C4,N
-ignore = E203,E501,W503,WPS111,WPS201,WPS202,WPS210,WPS211,WPS213,WPS219,WPS220,WPS221,WPS216,WPS226,WPS231,WPS232,WPS305,WPS336,WPS432,I001,I003,I004,I005
+ignore = E203,E501,W503,WPS111,WPS201,WPS202,WPS210,WPS211,WPS213,WPS219,WPS220,WPS221,WPS216,WPS226,WPS231,WPS232,WPS305,WPS336,WPS432,I001,I003,I004,I005,BLK100
```

Comment:
When I added the `canu_version = pkg_resources.get_distribution('canu').version` lines the `flake8-black` plugin began complaining `BLK100 Black would make changes.`. I couldn't figure out why, I assume it is because the value is going to change after the flake8 run (e.g. `canu_version` isn't a constant or something).

This just ignores that error for the linter.

Reply:
If you scroll to the far right you'll see the line isn't being removed, it's just appending `BLK100`.